### PR TITLE
#154 Create revision component

### PR DESCRIPTION
**Pull request by fkroockmann**
- Create a popin which lists all revision was saved . It allows you to make the choice about save.
- Add validate management

### CODE REVIEW

**Comment by ndufreche** [comment]
:+1: good job

**Comment by fkroockmann** [comment]
Thanks for review, you can merge :)

File: src/tb/apps/contribution/templates/index.twig [hunk 1]
```diff
@@ -50,6 +50,11 @@
             </button>
         </p>
         <p>
+            <button id="bundle-toolbar-global-validate" type="button" class="btn btn-default-grey btn-sm btn-block">
+                <i class="fa fa-floppy-o"></i> Valider
```
Comment (mickaelandrieu) [inline src/tb/apps/contribution/templates/index.twig]: Please, make a PR now to translate this one in **english**

File: src/tb/component/revisionselector/main.js [hunk 2]
```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2011-2013 Lp digital system
```
Comment (mickaelandrieu) [inline src/tb/component/revisionselector/main.js]: Did you notice we are in 2015 ? ^-^

File: src/tb/component/revisionselector/managers/Draft.js [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+                    keys = Object.keys(draft),
+                    subDraft,
+                    subDrafts = [],
+                    currentParent;
+
+                if (margin === undefined) {
+                    margin = 0;
+                }
+
+                for (i in draft) {
```
Comment (mickaelandrieu) [inline src/tb/component/revisionselector/managers/Draft.js]: what is "i" ?

File: src/tb/component/revisionselector/templates/tree.twig [hunk 4]
```diff
@@ -0,0 +1,12 @@
+<p class="dialog-title">Confirm saving changes content below :</p>
+<div class="bb5-list-data-toolbar bb5-toolbar-order">
+    <div class="row">
+        <p class="bb5-toolbar-order-col col-bb5-60"><a href="#">Title <i class="fa fa-angle-down"></i></a></p>
+        <p class="bb5-toolbar-order-col col-bb5-20"><a href="#">Autor <i class="fa fa-angle-down"></i></a></p>
```
Comment (mickaelandrieu) [inline src/tb/component/revisionselector/templates/tree.twig]: Author.

**Comment by mickaelandrieu** [comment]
@fkroockmann can you make another Pull request related to my comments please ?

Thanks.

**Pull request merged**